client/jetpack-cloud/sections/partner-portal/utils.ts

```diff
@@ -167,3 +167,13 @@ export function formatApiPartner( partner: APIPartner ): Partner {
 		} ) ),
 	};
 }
+
+/**
+ * Format the string by removing Jetpack, (, ) from the product name
+ *
+ * @param product Product name
+ * @returns Product title
+ */
+export function getProductTitle( product: string ): string {
+	return product.replace( 'Jetpack ', '' ).replace( '(', '' ).replace( ')', '' );
+}
```

Comment:
Good call moving this into a function.

Reply:
I don't think this will be a problem, but a note for posterity here: `String.replace` when called with string arguments will only replace the first instance of the target. Regex replacement arguments have no such restriction, though, if global mode is enabled. For example:

```
'Bananarama'.replace('an', 'ol');  // returns 'Bolanarama'
'Bananarama'.replace(/an/, 'ol');  // returns 'Bolanarama'
'Bananarama'.replace(/an/g, 'ol'); // returns 'Bololarama'
```

Reply:
I just moved this function from the component to the utils file, so missed this. Thanks for the suggestion. I have updated this.